ui/src/utils/cluster.utils.ts

```diff
@@ -20,9 +20,5 @@ export function generateClusterName(username = ''): string {
   const nameArray = [userPart, datePart, randomPart].filter(Boolean);
 
   // combine the 3 parts, truncate it at 28 characters, and remove a trailing '-', if any.
-  // Truncation is needed to allow generating wildcard certificates for OpenShift using Let's Encrypt: since LE insists
```

Comment:
Infra does not use LE for openshift certs.

Reply:
You'll need to merge the changes from #1016 into your branch (currently a merge conflict). When you did that, please update the comment here :) Probably the OCP part is not valid any longer.

Reply:
I kept your comment but removed the openshift 4 demo part.